File: main/a-very-big-sum/a-very-big-sum.cpp
#include <iostream>

int main()
{
    auto sum = 0LL;
    
    auto n = 0;
    for (std::cin >> n; n > 0; --n) {
        auto k = 0LL;
        std::cin >> k;
        sum += k;
    }

    std::cout << sum << '\n';
}

File: main/a-very-big-sum/a-very-big-sum.cs
using System;
using System.Linq;

internal static class Solution {
    private static void Main()
    {
        var n = int.Parse(Console.ReadLine());
        var s = Console.ReadLine().Split(' ').Select(long.Parse).Take(n).Sum();
        Console.WriteLine(s);
    }
}
